.terraform-docs.yaml

```diff
@@ -0,0 +1,33 @@
+formatter: "markdown"
```

Comment:
terradoc is already run as part of the pre-commit hooks with configuration common from https://github.com/terraform-ibm-modules/common-dev-assets/tree/8405107509b351b346887674d06ee37e98140493 , so this file is likely not needed here
Is this file necessary?

Reply:
done, I agree.